NRLMMD-GEOIPS/.github
# v1.6.0 .github - VERSION RELEASE - update templates

NOTE: This started as v1.5.4 release, but v1.6.0 justified due to data_fusion support updates.

# Related Issues
fixes NRLMMD-GEOIPS/geoips#51

# Reviewer Instructions
* Please confirm most recent completed pull request is labeled "DEVELOPMENTAL UPDATE" or "VERSION RELEASE":
    * https://github.com/NRLMMD-GEOIPS/.github/pulls?q=is%3Apr+is%3Aclosed
* Review "Files changed" tab
* Confirm appropriate testing was completed for these updates.

# Summary
Changes for version 1.6.0 update on repo .github.
See NRLMMD-GEOIPS/geoips#51 for additional repositories included in this update.

# Testing Instructions
None of the functionality within the .github repository is actually live until it is merged to main - so we'll just have to wait and see.

# Post Merge Steps
Once this PR has been merged, v1.6.0 will be tagged/released on the .github repo main branch.
